common/src/gas_provider/node.rs

```diff
@@ -238,6 +272,15 @@ impl<ExternalId: Clone, Id: Clone + Copy, Balance: Zero + Clone + Copy>
         matches!(self, Self::ReservedLocal { .. })
     }
 
+    /// Returns wether the node is of `Reserve` type
+    pub(crate) fn is_reserved(&self) -> bool {
+        matches!(self, Self::Reserved { .. })
+    }
+
+    pub(crate) fn is_orphan(&self) -> bool {
```

Comment:
I think it's not representative name, because `GasNode::External` can be considered as orphan as well